apps/coscrad-frontend/src/components/tenant/tenant-logo.tsx

```diff
@@ -6,10 +7,13 @@ export const TenantLogo = (): JSX.Element => {
     const { organizationLogoUrl } = useContext(ConfigurableContentContext);
 
     return (
-        <StyledImage
-            sx={{ width: { xs: '40px', sm: '55px', md: '61px' } }}
-            src={organizationLogoUrl}
-            alt="COSCRAD Logo"
-        />
+        //TODO parent organization url here
+        <IconButton>
```

Comment:
I’d be interested to see if this is used outside the footer. It seems like you want to include the tenant Icon, and provide social media links, and provide additional links with custom ICON URLs. This is because 
- this tenant logo probably serves additional purposes
- Blake has 2 internal links
    - TNG language 
    - TNG
 - social media uses built-in icon artwork and does not require a link